packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -79,6 +80,15 @@ describeWithDOM('mount', () => {
       mount(<div ref={spy} />);
       expect(spy).to.have.property('callCount', 1);
     });
+
+    it('should call trackWrapper', () => {
+      const spy = sinon.spy();
+      const originalTrackWrapper = wrapperSandbox.trackWrapper;
```

Comment:
eesh, this is a really brittle test that will definitely break in babel 7 and when node ships native ESM support. is this necessary? can we instead test the actual behavior?